src/Halogen/Svg/Attributes/Path.purs

```diff
@@ -0,0 +1,128 @@
+module Halogen.Svg.Attributes.Path
+  ( PathCommand -- Constructor not exported. Use m, l, h, v, c, s, q, t, a, or z
+  , CommandPositionReference(Abs, Rel)
+  , CommandArcChoice(Arc0, Arc1)
+  , CommandSweepChoice(Sweep0, Sweep1)
```

Comment:
Can all of these use the `(..)` (except `PathCommand`)

Reply:
Fixed with 269c75f05b9ed91f07baa020041933dd549c21b8